react-app/src/redux/game/reducer.js

```diff
@@ -0,0 +1,35 @@
+import { actionNames } from './actions';
+
+const initialState = {
+  history: [{ squares: Array(9).fill(null) }],
+  stepNumber: 0,
+  xIsNext: true,
+  status: ''
+};
+
+export const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case actionNames.jumpTo:
+      return {
+        ...state,
```

Comment:
This is great. FYI, you can also do `state.merge({ differences here })`

Reply:
Could you link me an example of how to do this?

Reply:
In this case you would do 
`return state.merge({stepNumber: action.stepNumber, xIsNext: action.xIsNext})`